.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,59 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ develop, master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ develop ]
+  schedule:
+    - cron: '40 11 * * 5'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
```

Comment:
what's that?

Reply:
https://docs.github.com/en/rest/reference/code-scanning#list-code-scanning-alerts-for-a-repository my understanding is you need to be in the security group to read these. Again auto generated from github.